contracts/strategies/pair/PairBasedStrategyLib.sol

```diff
@@ -278,36 +279,36 @@ library PairBasedStrategyLib {
   /// @param price Current price
   /// @return needToChange A boolean indicating if the fuse status should be changed
   /// @return status Exist fuse status or new fuse status (if needToChange is true)
-  function needChangeFuseStatus(FuseStateParams memory fuse, uint price) internal pure returns (
+  function needChangeFuseStatus(FuseStateParams memory fuse, uint price, uint poolPrice) internal pure returns (
```

Comment:
please add comment 
@param poolPrice Current price in the pool

also we need to modify comment to price
@param price Current price in the oracle